modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -127,7 +127,9 @@ export default class LargeVideoManager {
             // the video was not rendered, before the connection has failed.
             const isHavingConnectivityIssues
                 = APP.conference.isParticipantConnectionActive(id) === false;
-            if (isHavingConnectivityIssues
+            const isInLastN
+                = APP.conference.isLocalId(id) || APP.conference.isInLastN(id);
+            if ( (isHavingConnectivityIssues || !isInLastN)
```

Comment:
I was thinking that it's important to check in the case we pin a participant from the UI. However the bridge responds pretty quickly to our pin with a last N change event, but in theory it can take time. What will happen here with the latest changes we're planning in lib-jitsi-meet is have a different state for connectivity issues that will take care of the state between last N changed event and having actual video for the active/pinned participant on the large screen.